test/powershell/Modules/PowerShellGet/PowerShellGet.Tests.ps1

```diff
@@ -90,7 +90,7 @@ function Initialize
 
 function Remove-InstalledModules
 {
-    Get-InstalledModule -Name $ContosoServer -AllVersions -ErrorAction SilentlyContinue | Uninstall-Module -Force
+    Get-InstalledModule -Name $ContosoServer -AllVersions -ErrorAction SilentlyContinue | PowerShellGet\Uninstall-Module -Force
```

Comment:
There should not be any errors in uninstalling the test module, otherwise other test cases will fail.